### ISSUE TITLE
mia crash if we click to "Not Save" button 

### ISSUE THREAD

**Issue by servoz** [populse/populse_mia#96]
**Describe the bug**
If we make a change in the Data Browser tab and then switch to the Pipeline Manager tab, a pop-up window opens with the following message "Unsaved modifications in the Data Browser ! There are unsaved modifications in the database, you need to save or remove them before working with pipelines."  If we don't want to save the modification and so we click on the "Not Save" button and then we go back to the Data Browser tab, mia crash ...

**Generated error**
```
Traceback (most recent call last):
  File "/home/econdami/Git_Projects/populse_mia/python/populse_mia/user_interface/main_window.py", line 966, in tab_changed
    self.data_browser.table_data.add_columns()
  File "/home/econdami/Git_Projects/populse_mia/python/populse_mia/user_interface/data_browser/data_browser.py", line 934, in add_columns
    self.update_colors()
  File "/home/econdami/Git_Projects/populse_mia/python/populse_mia/user_interface/data_browser/data_browser.py", line 2095, in update_colors
    COLLECTION_CURRENT, tag).origin ==
AttributeError: 'NoneType' object has no attribute 'origin'

```

**To Reproduce**
1. Go to Data Browser tab, with data already loaded
2. Add a new tag 
3. Switch to Pipeline Manager tab
4. Click to the Not Save button
5. Switch to Data Browser tab
6. Observe the crash ...

**Expected behavior**
No mia crash, never, ever !

**Desktop (please complete the following information):**
 - OS: Linux 4.18.19-100.fc27.x86_64 #1 SMP Wed Nov 14 22:04:34 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
 - Populse_MIA version 1.2.1

**Comment by M40V** [comment]
We managed a quick [fix](https://github.com/populse/populse_mia/commit/0a8536eebdba1a9e6e5280243ee9f082d293a324) by modifying the conditions in _update_colors_ to detect Nonetype and act accordingly. 
The core of the issue seems to be relative to populse_db, either from the COLLECTION definition or from the rollback function to unsave modifications.
Currently I was not able to detect another case where not saving when switching tabs will cause a crash, but I may have forgotten a few.
I tried these modifications independently which caused no issues :
- Add a tag
- Remove a tag
- Modify a cell
- Remove documents
- Add documents
- Sort documents
